### PR TITLE
Xp7 support

### PR DESCRIPTION
**Pull request by cibietici**
This feature branch includes the change to support applications developed in XP7.0. Since this is a major change and we still need the previous master to support XP6.15 the version is set to 2.0.0 and we should make a new release from this branch.

### CODE REVIEW

File: gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -2,4 +2,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.3.1-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.6-bin.zip
```
Comment (tajakobsen) [inline gradle/wrapper/gradle-wrapper.properties:5]: Why is this only bumped to 5.6? The official ["library starter" bumps it to 6.4](https://github.com/enonic/starter-lib/blob/master/gradle/wrapper/gradle-wrapper.properties#L6) for XP7.

**Review by tajakobsen — CHANGES_REQUESTED** [review]
Good job! :)

But the README.md file should be updated with information about which versions to use with XP6 and which with XP7.

**Review by tajakobsen — APPROVED** [review]
Looks great! Good job! :+1:

**Review by tommytusj — APPROVED** [review]

**Review by tajakobsen — APPROVED** [review]
Good job!

**Review by nitelite — APPROVED** [review]

**Review by tommytusj — APPROVED** [review]

**Pull request merged**